### ISSUE TITLE
Does this also support t480?

### ISSUE THREAD

**Issue by amosbird** [linusyang92/macOS-ThinkPad-T480s#18]
I'd like to try out this promising config but am a bit worried about the compatibility of t480.

**Comment by kekkokk** [comment]
yeah it works, only thing is the trackpad kext, don't use the ApplePS2SmartTouchPad
